apps/portal/packages/app/src/styles.css

```
@@ -0,0 +1,584 @@
+/* Import Fonts (Alternative to index.html) */
+@import url('https://fonts.googleapis.com/css2?family=Inter:wght@400;700&family=JetBrains+Mono:wght@400;700&display=swap');
```

Stylelint (17.5.0)

[error] Expected "url('https://fonts.googleapis.com/css2?family=Inter:wght@400;700&family=JetBrains+Mono:wght@400;700&display=swap')" to be "'https://fonts.googleapis.com/css2?family=Inter:wght@400;700&family=JetBrains+Mono:wght@400;700&display=swap'" (import-notation)

(import-notation)